### DISCUSSION TITLE
Ways to detect an object is in sleep ?

### DISCUSSION THREAD

**Discussion by htgmanics** [pmndrs/use-cannon#108] [General]
In cannon.js, there are 2 events that will detect whether an object is sleeping:
```
sphereBody.addEventListener("sleepy",function(event){
    console.log("The sphere is feeling sleepy...");
});

sphereBody.addEventListener("sleep",function(event){
    console.log("The sphere fell asleep!");
});
```
Are there any ways to do that in use-cannon ?

**Comment by drcmda** [comment]
there should be, otherwise it should be easy to add. @codynova do we have this?

**Reply by codynova** [reply]
Nope, that's not in `use-cannon` yet, but would be easy enough to add. I've been thinking a lot lately about whether there is a more generic API that we could adopt that would prevent us from having to manually implement all the worker data bridging...

**Reply by htgmanics** [reply]
Thanks for the reply. 
Can you advice me on how to implement it manually ?
